cinemy/login/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import path
 from login.views import signup, auth, pre_signup
 from django.contrib.auth.views import LogoutView
+from rest_framework.authtoken import views
 
 urlpatterns = [
     path("login/", auth, name="login"),
     path("logout/", LogoutView.as_view(), name="logout"),
     path("pre_signup/", pre_signup, name="pre_signup"),
     path("signup/<slug:uidb64>/", signup, name="signup"),
+    path("generate_api_token/", views.obtain_auth_token),
```

Comment:
This view will generate "9944b09199c62bcf9418ad846dd0e4bbdfc6ee4b" like tokens.
For extra protection, the task asks to encode this token in base64 as well.

Reply:
Will modify the code to generate the token in base64 format.